javascript/javascript1/week3/jsweek3.js

```diff
@@ -0,0 +1,172 @@
+//Item array removal
+let names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+names = names.filter(function (item) {
+  return item !== nameToRemove;
+});
+// Code done
+console.log(names);
+// ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//******************************
+//  When will we be there??
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+function ArrivalTime(AaTe) {
+  let numValue = (AaTe.destinationDistance / AaTe.speed) * 60; // getting number of minutes
+  let hours = Math.floor(numValue / 60); // getting hours
+  let minutes = Math.floor(numValue % 60); //getting minutes as an integer
+  return hours + " hours and " + minutes + " minutes";
+}
+
+const travelTime = ArrivalTime(travelInformation);
+console.log(travelTime); // 8 hours and 38 minutes
+
+//******************************
+// Save a note
+
+const notes = [];
+
+function saveNote(content, id) {
+  // write some code here
+  notes.push({ content: content, id: id });
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+// Get a note
+
+function getNote(id) {
+  // your code here
+  if (isNaN(id)) {
+    console.log("please enter a vild number");
+    return;
+  }
+  let output = "error the value not exist";
+
+  for (let i = 0; i < notes.length; i++) {
+    if (id === notes[i].id) {
+      output = notes[i];
+      break;
+    }
+  }
+  console.log(output);
+}
+
+const firstNote = getNote(1); // {content: 'Pick up groceries', id: 1}
+
+//Log out notes
+
+function logOutNotesFormatted() {
+  // your code here
+
+  for (let i = 0; i < notes.length; i++) {
+    console.log(
+      "The note with id: ",
```

Comment:
Great, but maybe using string interpolation also here would make the code more concise  🦸‍♂️